Pod/Classes/VOKCoreDataCollectionTypes.h

```diff
@@ -0,0 +1,61 @@
+//
+//  VOKCoreDataCollectionTypes.h
+//  Pods
+//
+//  Created by Carl Hill-Popper on 10/23/15.
+//
+//
+
+#ifndef VOKCoreDataCollectionTypes_h
```

Comment:
Actually it does seem common for languages with generics.  Maybe they should stay.